apps/codeforafrica/src/components/Projects/Projects.js

```diff
@@ -57,7 +55,9 @@ const Projects = React.forwardRef(function Projects(
   }, [data]);
 
   useEffect(() => {
-    router.push(queryParams, undefined, {
+    const [pathname] = router.asPath.split("?");
```

Comment:
Why are we using `asPath` here instead of `pathname`? If we need to use asPath, please see the warning [here](https://nextjs.org/docs/pages/api-reference/functions/use-router#router-object).

Reply:
@kilemensi pathname exposes is `/[...slugs]` and comes with this error.
![image](https://github.com/CodeForAfrica/ui/assets/39160236/b4e16327-1c5f-42b7-8c8b-e08ccbf8372c)

Reply:
👍🏽  @koechkevin... then if you _have_ to use `asPath`, use is as directed in the docs.